lld/test/ELF/dtlto/index.test

```diff
@@ -0,0 +1,43 @@
+# REQUIRES: x86
```

Comment:
This is quite similar to imports.test and can be merged into it. empty.test can be removed if it does not add additional coverage.

Reply:
I have pushed a commit to combine the coverage in `imports.test`, `index.test` and `save-temps.test` into one test: `files.test`.